### ISSUE TITLE
Port Lootr to 1.17.1: waiting on Mixins/mods awaiting 1.17.1

### ISSUE THREAD

**Issue by noobanidus** [noobanidus/Lootr#46]
While it's possible to run Mixins's beta/alpha release within dev, it's not possible to release this until Mixins have been merged into Forge.

**Awaiting 1.17.1**:

- Twilight Forest (#32)
